src/main.js

```diff
@@ -97,46 +97,88 @@ var descriptors = [
 
 
 
-
-
 // We've provided a few variables below
 var savedCovers = [
   new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
 ];
 var currentCover;
 
+var makeMyBookButton = document.querySelector('.create-new-book-button');
+
+
+
+makeMyBookButton.addEventListener('click', function(event){
+  var userCoverURL = document.querySelector('#cover');
+  var userTitle = document.querySelector('#title');
+  var userDescriptor1 = document.querySelector('#descriptor1');
+  var userDescriptor2 = document.querySelector('#descriptor2');
+  var uniqueCoverInput = new UniqueCover(userCoverURL.value, userTitle.value, userDescriptor1.value, userDescriptor2.value);
+  console.log(uniqueCoverInput);
+  toggleToHome();
+  document.querySelector('.cover-image').src = uniqueCoverInput.coverURL;
+  document.querySelector('.cover-title').innerText = uniqueCoverInput.title;
+  document.querySelector('.tagline-1').innerText = uniqueCoverInput.descriptor1;
+  document.querySelector('.tagline-2').innerText = uniqueCoverInput.descriptor2;
+  event.preventDefault();
+});
```

Comment:
This looks fantastic. 

Let's definitely play with refactoring the spaces and omitting the console.logs at the end of the project.